### DISCUSSION TITLE
[This forum is public!] 👋 How to Use Paste Discussions 👋

### DISCUSSION THREAD

**Discussion by corinne-nagel** [twilio-labs/paste#1203] [General]
Welcome to Paste Discussions!

### For dependencies and feature requests:

Submit your request here: https://airtable.com/shrvMvL3BG6Zm0f9l This form is for Twilio employees only. If you're a customer of ours, please start a Discussion in the ways listed below!

### Here are some things you can do here:

**🙏  Q&A:** Ask us how to integrate Paste, how to use a component/pattern, or questions about the accessibility of your work.

**🏗  Start the contribution process:** Working on a new component or pattern idea? Share it here to get the process started. Learn more by reading the [Pattern contribution guidelines](https://paste.twilio.design/patterns/contribute/).

**💡 Share ideas or feature requests:** Have an idea on how to make Paste better? We’d love to hear about it!

**😍  Show ’n tell:** Building something exciting using Paste? We’d love to see 👀 !

If you think something is a bug, please [report it here](https://github.com/twilio-labs/paste/issues).

### Some tips for posting:

**Check out the docs:** Please [read the docs](https://paste.twilio.design/) thoroughly to understand what is available, what isn’t, and how to build

**Check that you're on the latest Paste version:** We might have fixed an issue you're noticing already! Make sure you're on the most recent Paste version. You can check the latest versions for Core, component, and library packages on the [Core page](https://paste.twilio.design/core).

**Search:** Use the search to look for existing discussions or related issues. If possible, link to discussions in your posts.

**Build:** If possible, share your work in a CodeSandbox using the [Paste starter kit](https://codesandbox.io/s/paste-starter-kit-rj7yy) or in a Figma file so we can easily help you.

**Context:** Provide as much information as you can to avoid too much back and forth. You’re much more familiar with your product than we are! Let us know what product you work on, who your customers are, what research you’ve already done, and what constraints you’re facing.

**Privacy:** In some cases, your team might not be able to share information on features and roadmaps in this public forum. If this is the case, please sign up for [office hours](https://docs.google.com/spreadsheets/d/10vD91zdwr9hjgDOV3HSGnoC0LgvYw-thmn3X_kvW0xk/edit#gid=800758837) (Twilio employees only) and/or generalize your questions in Github Discussions. You can also share a Figma link, and we can continue the conversation in Figma, office hours, and Slack.

Follow our [Code of Conduct](https://github.com/twilio-labs/.github/blob/01528a2f1b990842f8bd3ad0cd254d1028bdc28d/CODE_OF_CONDUCT.md).

💛  the Paste team

**Comment by wilfredomerced** [comment]


**Reply by serifluous** [reply]
Hey @wilfredomerced , thanks for posting! Can you start a new discussion thread with this question?

Go here: https://github.com/twilio-labs/paste/discussions and click on the "New discussion" button. Thanks!

**Reply by wilfredomerced** [reply]
ah thanks! will delete this post
